css/src/atomics/colors.scss

```diff
@@ -0,0 +1,225 @@
+// Framework colors
+
+@each $name, $color-set in $colors {
+	$base: nth($color-set, $color-index-base);
+	$background: nth($color-set, $color-index-background);
+	$dark: nth($color-set, $color-index-dark);
+	$hover: nth($color-set, $color-index-hover);
+	$active: nth($color-set, $color-index-active);
+	$invert: nth($color-set, $color-index-invert);
+
+	.color-#{$name} {
+		color: $base !important;
+	}
+
+	a.color-#{$name} {
+		&:hover,
+		&.focus-visible {
+			color: $hover !important;
+		}
+
+		&:active {
+			color: $active !important;
+		}
+	}
+
+	.color-#{$name}-invert {
+		color: $invert !important;
+	}
+
+	.background-color-#{$name} {
+		outline-color: $invert;
+		background-color: $base !important;
+	}
+
+	.background-color-#{$name}-tablet {
+		@include tablet {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-desktop {
+		@include desktop {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-widescreen {
+		@include widescreen {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-invert {
+		outline-color: $base;
+		background-color: $invert !important;
+	}
+
+	.background-color-#{$name}-light {
+		outline-color: $dark;
+		background-color: $background !important;
+	}
+
+	.background-color-#{$name}-dark {
+		outline-color: $background;
+		background-color: $dark !important;
+	}
+}
+
+// Text color
+
+.color-text {
+	color: $text !important;
+}
+
+.color-text-subtle {
+	color: $text-subtle !important;
+}
+
+.color-text-overlay-invert {
+	color: $overlay-invert;
+}
+
+.color-current-color {
+	color: currentColor !important;
+}
+
+.fill-current-color {
+	fill: currentColor !important;
+}
+
+.outline-color-text {
+	outline-color: $text !important;
+}
+
+.stroke-current-color {
+	stroke: currentColor !important;
+}
+
+// Background color
+
+.background-color-body {
+	outline-color: $text;
+	background-color: $body-background !important;
+}
+
+.background-color-body-tablet {
```

Comment:
Could we quickly audit the how many usages of body-background classes we have already with screensizes prepended? Is it necessary that we include these?

Reply:
has-body-background-mobile: 3
has-body-background-dark-tablet: 1

Shall we keep these two?

Reply:
@v-mingho Let's remove them for now. We can include them later if it really become necessary.

Reply:
Removed